src/api/bucket_api.js

```diff
@@ -1439,7 +1442,8 @@ module.exports = {
                 },
                 caching: {
                     $ref: 'common_api#/definitions/bucket_cache_config'
-                }
+                },
+                should_create_uls: { type: 'boolean' } // should create underlying storage
```

Comment:
I dont think this abbrev makes sense because it is impossible to know what it means unless you are the one who invented it... I would keep the it long instead

Reply:
sure